src/routes/tokens/components/ApiToken.tsx

```diff
@@ -0,0 +1,178 @@
+                                fontSize="sm"
+                                opacity={isHovered ? '100%' : '50%'}
+                            >
+                                {`at ${apiToken.created_at.toLocaleTimeString()}`}
+                            </Text>
+                        </HStack>
+
+                        {apiToken.expires_at === null ? (
+                            <Text
+                                fontSize="sm"
+                                fontWeight="400"
+                            >{`Never Expires`}</Text>
+                        ) : (
+                            <HStack
+                                color={tokenIsExpired ? 'red.500' : undefined}
+                                spacing="1.5"
+                                fontSize="sm"
+                                fontWeight="400"
+                            >
+                                <Text>
+                                    {'Expire' +
+                                        (tokenIsExpired ? 'd' : 's') +
+                                        ` on ${
+                                            shortFormatDate(
+                                                apiToken.expires_at.toDateString()
+                                            ).split(',')[0]
+                                        } `}
+                                </Text>
+                                <Text opacity={isHovered ? '100%' : '50%'}>
+                                    {` at ${apiToken.expires_at.toLocaleTimeString()}`}
+                                </Text>
+                            </HStack>
+                        )}
+                    </VStack>
+                    <Spacer />
+                    <Center width="120px">
+                        <Button
+                            onClick={onConfirmOpen}
+                            variant="outline"
+                            size="sm"
+                            fontWeight="600"
+                        >
+                            Delete
+                        </Button>
+
+                        <AlertDialog
+                            isOpen={isConfirmOpen}
+                            leastDestructiveRef={cancelRef}
+                            onClose={onConfirmClose}
+                            size="sm"
+                        >
+                            <AlertDialogOverlay>
+                                <AlertDialogContent fontSize="sm">
+                                    <AlertDialogHeader
+                                        fontSize="lg"
+                                        fontWeight="bold"
+                                    >
+                                        Delete token
+                                    </AlertDialogHeader>
+                                    <AlertDialogBody>
+                                        Are you sure you want to delete this
+                                        token?
```

Comment:
Maybe we could expand the text to explain that deleting the token means every script that use it will stop working. A user might think "delete the token" would only prevent using it for new scripts.

(I don't really know, I try to put myself in the shoes of a data scientist but it's not easy)